src/OpenLoco/Platform/Crash.h

```diff
@@ -0,0 +1,3 @@
+#pragma once
+using CExceptionHandler = void*;
+CExceptionHandler crash_init();
```

Comment:
Is there any reason to save the exception handler? Its never used. May as well return void.

Reply:
Dropped the result

Reply:
Actually I will restore it and release after we're done with it. There is some cleanup it performs:

https://github.com/wincent/Google-Breakpad/blob/7b873221e6741208827b2a4a5f2dda35ceccb474/src/client/windows/handler/exception_handler.cc#L223-L305